app/uk/gov/hmrc/pensionschemereturn/controllers/PsrOverviewController.scala

```diff
@@ -49,12 +51,14 @@ class PsrOverviewController @Inject() (
     fromDate: String,
     toDate: String
   ): Action[AnyContent] = Action.async { implicit request =>
+    val effectiveFromDate = Seq(fromDate, config.earliestPsrPeriodStartDate).max
+
     val Seq(srnS) = requiredHeaders("srn")
     authorisedAsPsrUser(srnS) { _ =>
       logger.debug(
-        s"Retrieving overview - with pstr: $pstr, fromDate: $fromDate, toDate: $toDate"
+        s"Retrieving overview - with pstr: $pstr, fromDate: $effectiveFromDate, toDate: $toDate"
```

Comment:
Maybe we could display both the date from the input parameter and the effective date here (and in versions calls), so it is obvious for debugging. E.g.:
`fromDate: 1900-04-06 (effective: 2021-04-06), toDate: 2024-04-05] `

Reply:
this is updated now